common/src/service/CommonConstants.cs

```diff
@@ -117,6 +117,9 @@ static class CommonConstants
         public const string IotHubErrorCode = "IotHubErrorCode";
         public const string HttpErrorCodeName = "iothub-errorcode";
 
+        // Custom HTTP response contents
+        public const string ErrorCode = "errorCode";
```

Comment:
Does this need to be public? How about just internal?

Reply:
Good point. Changing to internal.